index.js.flow

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+type TemplateStringsArray = $ReadOnlyArray<string>;
+
+export type Level = $Values<{
+	None: 0,
+	Basic: 1,
+	Ansi256: 2,
+	TrueColor: 3
+}>;
+
+export type ChalkOptions = {|
+	enabled?: boolean,
+	level?: Level
+|};
+
+export type ColorSupport = {|
+	level: Level,
+	hasBasic: boolean,
+	has256: boolean,
+	has16m: boolean
+|};
+
+export interface Chalk {
+	(...text: string[]): string,
+	(text: TemplateStringsArray, ...placeholders: string[]): string,
+	constructor(options?: ChalkOptions): Chalk,
+	enabled: boolean,
+	level: Level,
+	rgb(r: number, g: number, b: number): Chalk,
```

Comment:
Can you not use `this` instead of `Chalk` here? I'm not that familiar with Flow. Just checking. Like: https://github.com/chalk/chalk/blob/58483b5ab110775eb0cfc4999f1cf7e971a00fda/types/index.d.ts#L34

Reply:
Not in an interface type https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBLAdgFwKYBOUAhgMZ5gDCAFsTANZgDeqYYBA5gEYAUBAXGCwBXALZdCAGjAdBI8VLBc5YiQQCUgnNQwBnANyoAvuiA